scripts/post-uninstall-rpm.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+
+# This check works only for RPMs
+if [ $1 -ge 1 ]; then
+    # Package upgrade, do not uninstall
+    exit 0
+fi
+
+# Delete the link to the binary
+if type update-alternatives >/dev/null 2>&1; then
+    update-alternatives --remove "httptoolkit" "/opt/HTTP Toolkit/httptoolkit"
+else
+    rm -f "/usr/bin/httptoolkit"
```

Comment:
If `update-alternatives` is not installed, the `/usr/bin/httptoolkit` is removed directly. The %post script created by electron-builder does the same thing on installs and upgrades:

```bash
#!/bin/bash

if type update-alternatives 2>/dev/null >&1; then
    # Remove previous link if it doesn't use update-alternatives
    if [ -L '/usr/bin/httptoolkit' -a -e '/usr/bin/httptoolkit' -a "`readlink '/usr/bin/httptoolkit'`" != '/etc/alternatives/httptoolkit' ]; then
        rm -f '/usr/bin/httptoolkit'
    fi
    update-alternatives --install '/usr/bin/httptoolkit' 'httptoolkit' '/opt/HTTP Toolkit/httptoolkit' 100 || ln -sf '/opt/HTTP Toolkit/httptoolkit' '/usr/bin/httptoolkit'
else
    ln -sf '/opt/HTTP Toolkit/httptoolkit' '/usr/bin/httptoolkit'
fi
```